cli/lib/kontena/cli/apps/restart_command.rb

```diff
@@ -27,10 +27,11 @@ def execute
     def restart_services(services)
       services.each do |service_name, opts|
         if service_exists?(service_name)
-          puts "restarting #{prefixed_name(service_name)}"
-          restart_service(token, prefixed_name(service_name))
+          ShellSpinner "sending restart signal to #{service_name.colorize(:cyan)} " do
+            restart_service(token, prefixed_name(service_name))
+          end
         else
-          puts "No such service: #{service_name}".colorize(:red)
+          STDERR.puts "WARNING: no such service: #{service_name}".colorize(:yellow)
```

Comment:
I'm not completely sold on the idea that "app" commands should exit with error if some service does not exist. At least these kind of changes should be done in a separate pr. @nevalla what do you think?